src/pudl/analysis/allocate_net_gen.py

```diff
@@ -1,103 +1,100 @@
-generation_eia923 or using fuel_consumed_mmbtu from boiler_fuel_eia923 because there are
-some small discrepancies between the total amounts of generation/fuel reported in these
-two tables.
-
-In plant-report_date's where NONE of the generators report more granular generation or fuel consumption,
-we create a generator record for each associated fuel type. Those records are merged with the
-generation_fuel_eia923 table on plant, prime mover code, and fuel type. Each group of
-plant, prime mover, and fuel will have some amount of reported net generation associated
-with it, and one or more generators. The net generation is allocated among the
-generators within the group in proportion to their capacity. Then the allocated net
-generation or fuel consumption is summed up by generator.
-
-In the hybrid case, where only SOME of of a plant's generators report the more granular
-generation or fuel data, we use a combination of the two allocation methods described above.
-First, the total generation or fuel consumption reported across a plant in the
-generation_fuel_eia923 table is allocated between the two categories of generators
-(those that report fine-grained generation/fuel data, and those that don't) in direct
-proportion to the fraction of the plant's generation which is reported in the
-generation_eia923 table (or the fraction of the plant's fuel consumption which is reported
-in the boiler_fuel_eia923 table), relative to the total generation or fuel reported in the
-generation_fuel_eia923 table.
-
-Known Drawbacks of this methodology:
+* The **ALL** Coverage Records: where ALL generators report in the granular tables.
+* The **NO** Coverage Records: where NONE of the generators report in the granular
+  tables.
+* The **SOME** Coverage Records: where only SOME of the generators report in the granular
+  tables.
+
+For the **ALL** generators, the data columns reported in the
+:ref:`generation_fuel_eia923` table are allocated in proportion to data reported in the
+granular data tables. We do this instead of directly using the data columns from the
+granular tables because we are assuming the :ref:`generation_fuel_eia923` table is the
+truth when there are some discrepencies between the granular tables and the
+:ref:`generation_fuel_eia923` table.
+
+For the **NONE** generators, the data columns reported in the
+:ref:`generation_fuel_eia923` table are allocated in proportion to the each generator's
+capacity.
+
+For the **SOME** generators, we use a combination of the two allocation methods
+described above. First, the data columns reported in the :ref:`generation_fuel_eia923`
+table is allocated between the two categories of generators (those that report granular
+data, and those that don't) in direct proportion to the fraction of the records the data
+which is reported in the granular tables relative to the data that is reported in the
+:ref:`generation_fuel_eia923` table.
```

Comment:
"First, the data columns reported in the generation_fuel_eia923 table are allocated between the two categories of generators: those that report granular data, and those that don't. The fraction allocated to each of those categories is based on how much of the total is reported in the granular tables. If T is the total reported, and X is the quantity reported in the granular tables, then the allocation is X/T to the generators reporting granular data, and (T-X)/T to the generators not reporting granular data. Within each of those categories the allocation then follows the **ALL** or **NONE** allocation methods described above."

Reply:
this is great ty!